src/cockpit/transports.py

```diff
@@ -412,11 +412,22 @@ class StdioTransport(_Transport):
     def __init__(self, loop: asyncio.AbstractEventLoop, protocol: asyncio.Protocol, stdin: int = 0, stdout: int = 1):
         super().__init__(loop, protocol, stdin, stdout)
 
+    def _close(self) -> None:
```

Comment:
I'd actually prefer we don't do this.

Reply:
OK, moved to SubprocessTransport and dropped the StdioTransport changes.